ui/src/app/components/notification-banner.spec.tsx

```diff
@@ -21,7 +24,7 @@ describe('NotificationBanner', () => {
   };
 
   it('should render', () => {
-    const wrapper = component();
-    expect(wrapper.exists()).toBeTruthy();
+    component();
+    expect(screen.getByText('blah')).toBeInTheDocument();
```

Comment:
where does blah come from?  Is there a const you can reference?

Reply:
"Blah" was coming from a hard-coded parameter value being passed in the `component` function within this test:
![image](https://github.com/all-of-us/workbench/assets/24514630/9ac6b84e-b515-4910-a7c7-932fa7f4f9e9)

Shifting this out to a shared const makes a lot of sense:
![image](https://github.com/all-of-us/workbench/assets/24514630/1d34923a-cf67-42a9-926d-3724b881b732)